conf/modules.config

```diff
@@ -806,6 +806,22 @@ process {
         ext.prefix = { "${meta.assembler}-CONCOCT-${meta.id}" }
     }
 
+    withName: COMEBIN_RUNCOMEBIN {
```

Comment:
We should probably set errorStrategy = "ignore" on this one?

Reply:
Will make the pipeline more robust in execution, yes, I like it.